governance_DRAFT.md

```diff
@@ -1,95 +1,206 @@
+through any legal means, though the Institutional Partner is obliged to inform
+the Steering Council of any such decisions. This could involve a non-profit
+organization raising money from private foundations and donors or a for-profit
+company building proprietary products and services that leverage Project
+Software and Services. Funding acquired by Institutional Partners to work on The
+Project is called Institutional Funding. However, no funding obtained by an
+Institutional Partner can override the decisions of the Steering Council. If a
+Partner has funding to do Project work and the Council decides to not pursue
+that work as a project, the Partner is free to pursue it on their own. However,
+in this situation, that part of the Partner’s work will not be under The Project
+umbrella.
 
 ## Changing the Governance Documents
 
-Changes to the governance documents are submitted via a GitHub pull request to The Project's governance documents GitHub repository at https://github.com/pangeo-data/governance. The pull request is then refined in response to public comment and review, with the goal being consensus in the community. The period for open comment will last in proportional to the complexity of the changes; major changes will be open for comment for no less than two weeks. After this open period, a Steering Council Member proposes to the Steering Council that the changes be ratified and the pull request merged (accepting the proposed changes) or proposes that the pull request be closed without merging (rejecting the proposed changes). The Member should state the final commit hash in the pull request being proposed for acceptance or rejection and briefly summarize the pull request. A minimum of 80% of the Steering Council must vote and at least 2/3 of the votes must be positive to carry out the proposed action (fractions of a vote rounded up to the nearest integer).
+Changes to the governance documents are submitted via a GitHub pull request to
+The Project's governance documents GitHub repository at
+https://github.com/pangeo-data/governance. The pull request is then refined in
+response to public comment and review, with the goal being consensus in the
+community. The period for open comment will last in proportional to the
+complexity of the changes; major changes will be open for comment for no less
+than two weeks. After this open period, a Steering Council Member proposes to
+the Steering Council that the changes be ratified and the pull request merged
+(accepting the proposed changes) or proposes that the pull request be closed
+without merging (rejecting the proposed changes). The Member should state the
+final commit hash in the pull request being proposed for acceptance or rejection
+and briefly summarize the pull request. A minimum of 80% of the Steering Council
+must vote and at least 2/3 of the votes must be positive to carry out the
+proposed action (fractions of a vote rounded up to the nearest integer).
```

Comment:
How do we resolve the catch-22 that, in order to adopt this governance document, we need a steering council to approve it? We did not make a plan for this.

Reply:
this is an active part of the discussion in #8. I'm going to merge these modifications without tying this issue to sorting this detail out.